packages/atomic-react/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "build": "npm run clean && npm run compile && npm run copy:assets",
-    "dev": "npm link ../samples/headless-react/node_modules/react",
```

Comment:
Why change the script name ? 🤔 

Might affect some other scripts in the repo.

Reply:
It's cleared as to what it does. I don't think it will affect other scripts, this is called when you run `dev` in the atomic-react samples